docs/reference/search/search-your-data/retrievers-overview.asciidoc

```diff
@@ -69,8 +68,9 @@ When using compound retrievers, only the query element is allowed, which enforce
 [[retrievers-overview-example]]
 ==== Example
 
-The following example demonstrates the powerful queries that we can now compose, and how retrievers simplify this process. We can use any combination of retrievers we want, propagating the
-results of a nested retriever to its parent. In this scenario, we'll make use of all 4 (currently) available retrievers, i.e. `standard`, `knn`, `text_similarity_reranker` and `rrf`.
+The following example demonstrates the powerful queries that we can now compose, and how retrievers simplify this process.
+We can use any combination of retrievers we want, propagating the results of a nested retriever to its parent.
+In this scenario, we'll make use of all 4 (currently) available retrievers, i.e. `standard`, `knn`, `text_similarity_reranker` and `rrf`.
```

Comment:
That's a good question. I think no for now - the reasoning being that rules are a niche use case, and as we have plans to add more retrievers it will soon become unwieldy to have one monster example. Instead, I'll reword to say that we're using some retrievers and that others are available.